### PR TITLE
Remove enumeration labels from reader workload

### PR DESCRIPTION
**Pull request by johnrchrdsn**
Remove enumerants from log names when logging actions and replace with
names suitable for being presented on charts.

### CODE REVIEW

**Comment by johnrchrdsn** [comment]
Potential fix for issue #84

File: wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java [hunk 1]
```diff
@@ -83,19 +83,16 @@ private void dismissWelcomeView() throws Exception {
     }
 
     private void openFile(final String filename) throws Exception {
-
-        String testTag = "openfile";
```
Comment (jimboatarm) [inline wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java:87]: You need to leave the testTag.  We parse the log files on this to form the groups, then the string below is used as the result name.

File: wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java [hunk 2]
```diff
@@ -83,19 +83,16 @@ private void dismissWelcomeView() throws Exception {
...
-
         // Replace whitespace and full stops within the filename
         String file = filename.replaceAll("\\.", "_").replaceAll("\\s+", "_");
 
-        timingResults.put(String.format(testTag + "_" + "1" + "_" + file), selectLocalFilesList());
```
Comment (jimboatarm) [inline wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java:92]: Leave the testTag

File: wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java [hunk 3]
```diff
@@ -83,19 +83,16 @@ private void dismissWelcomeView() throws Exception {
...
+        timingResults.put(String.format("local_files"), selectLocalFilesList());
 
         // On some devices permissions to access local files occurs here rather than the earlier step
         confirmAccess();
-        timingResults.put(String.format(testTag + "_" + "2" + "_" + file), selectSearchFileButton());
-        timingResults.put(String.format(testTag + "_" + "3" + "_" + file), searchFileList(filename));
-        timingResults.put(String.format(testTag + "_" + "4" + "_" + file), openFileFromList(filename));
+        timingResults.put(String.format("search_button"), selectSearchFileButton());
```
Comment (jimboatarm) [inline wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java]: will need to be (testTag + "search_button")

File: wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java [hunk 4]
```diff
@@ -152,17 +149,14 @@ private Timer openFileFromList(final String file) throws Exception {
     }
 
     private void gesturesTest(final String filename) throws Exception {
-
-        String testTag = "gestures";
```
Comment (jimboatarm) [inline wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java:156]: Same comments as above.  Need to keep testTag

File: wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java [hunk 5]
```diff
@@ -152,17 +149,14 @@ private Timer openFileFromList(final String file) throws Exception {
...
-
         // Perform a range of swipe tests at different speeds and on different views
         LinkedHashMap<String, GestureTestParams> testParams = new LinkedHashMap<String, GestureTestParams>();
-        testParams.put("1", new GestureTestParams(GestureType.UIDEVICE_SWIPE, Direction.DOWN, 200));
-        testParams.put("2", new GestureTestParams(GestureType.UIDEVICE_SWIPE, Direction.UP, 200));
-        testParams.put("3", new GestureTestParams(GestureType.UIOBJECT_SWIPE, Direction.RIGHT, 50));
-        testParams.put("4", new GestureTestParams(GestureType.UIOBJECT_SWIPE, Direction.LEFT, 50));
-        testParams.put("5", new GestureTestParams(GestureType.PINCH, PinchType.OUT, 100, 50));
-        testParams.put("6", new GestureTestParams(GestureType.PINCH, PinchType.IN, 100, 50));
+        testParams.put("swipe_down", new GestureTestParams(GestureType.UIDEVICE_SWIPE, Direction.DOWN, 200));
```
Comment (jimboatarm) [inline wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java]: will need to be (testTag + "swipe_down")

File: wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java [hunk 6]
```diff
@@ -180,7 +174,7 @@ private void gesturesTest(final String filename) throws Exception {
 
             UiObject view = new UiObject(new UiSelector().resourceId("com.adobe.reader:id/viewPager"));
 
-            String runName = String.format(testTag + "_" + pair.getKey());
```
Comment (jimboatarm) [inline wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java:183]: Need to keep the testTag

File: wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java [hunk 7]
```diff
@@ -207,16 +201,15 @@ private void gesturesTest(final String filename) throws Exception {
 
     private void searchPdfTest(final String filename, final String[] searchStrings) throws Exception {
 
-        String testTag = "search";
```
Comment (jimboatarm) [inline wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java:210]: Need to keep the testTag

**Pull request merged**